ioservice/io_foms.c

```diff
@@ -1945,8 +1946,8 @@ static int io_finish(struct m0_fom *fom)
 		}
 	}
 
-	M0_LOG(M0_DEBUG, "got    fom: %lu, req_count: %lx, "
-	       "count: %lx, nob: %lx", fom_obj->fcrw_fom_start_time,
+	M0_LOG(M0_DEBUG, "got    fom: %"PRIi64", req_count: %"PRIi64", "
+	       "count: %"PRIx64", nob: %"PRIx64"", fom_obj->fcrw_fom_start_time,
```

Comment:
I suggest to have:
```
"count: %"PRIi64", nob: %"PRIi64""
```

Otherwise some of these numbers are in HEX, and some in DEC.
This will be confusing/misleading.

Reply:
Let's keep the original formats, some scripts may depend on them.